app/Models/CategoryDAO.php

```diff
@@ -9,7 +9,7 @@ public function addCategory($valuesTmp) {
 		$stm = $this->bd->prepare($sql);
 
 		$values = array(
-			mb_strcut($valuesTmp['name'], 0, 255, 'UTF-8'),
+			mb_strcut($valuesTmp['name'], 0, 191, 'UTF-8'),
```

Comment:
Not a big fan of magic values. It would be better to have constants. Same comment goes for all magic values.

Reply:
Done in https://github.com/FreshRSS/FreshRSS/pull/2027/commits/d8a3c8cfb90c97b72f549c7ba13870ebf03249b5 (but not a big fan of the additional complexity of a constant in this case :-) )